app/controllers/dashboard_controller.rb

```diff
@@ -652,7 +652,7 @@ def session_init(db_user)
     @css.merge!(@settings[:display])
     @css.merge!(THEME_CSS_SETTINGS[@settings[:display][:theme]])
 
-    session[:user_TZO] = params[:user_TZO] ? params[:user_TZO].to_i : nil     # Grab the timezone (future use)
+    session[:user_TZO] = params[:user_TZO].try(:to_i)     # Grab the timezone (future use)
```

Comment:
I believe this expression used to evaluate to `nil` when `params[:user_TZO]` is `nil`.  Now it would evaluate to 0, since `nil.to_i` is 0.  Suggest reverting this to the old syntax unless you can think of something cleaner.